src/sass/style.scss

```diff
@@ -23,6 +23,8 @@
   font-style: normal;
 }
 
+@import "~cropperjs/dist/cropper.css";
```

Comment:
We should move this to the vic scss file.

Reply:
Thanks @jbalboni, this import has been moved to the vic scss file.